tests/model/range.js

```diff
@@ -593,20 +593,22 @@ describe( 'Range', () => {
 			expect( transformed[ 0 ].end.path ).to.deep.equal( [ 4, 7 ] );
 		} );
 
-		it( 'should stick to moved range, if the transformed range is collapsed #1', () => {
+		it( 'should not stick to moved range, if the transformed range is collapsed #1', () => {
```

Comment:
As I stated above, those tests were probably to meet 100% CC and we have to decide on something. I don't remember any situation where we did depend on this behaviour.

OTOH, when I think of it now I have some doubts, so I will rethink it.